src/app.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Feel free to debug with console.logs, but this rule is to ensure no logs get pushed into production

Reply:
Oh sorry I forgot to delete this line, thank you for the observation